brands/shop/funeral_directors.json

```diff
@@ -1,4 +1,18 @@
 {
+  "shop/funeral_directors|PFG": {
+    "countryCodes": ["fr"],
+    "match": [
+      "shop/funeral_directors|Pompes Funèbres Générales"
+    ],
+    "nocount": true,
+    "tags": {
+      "brand": "PFG",
+      "brand:wikidata": "Q3396087",
+      "brand:wikipedia": "fr:Pompes funèbres générales",
+      "name": "Pompes Funèbres Générales",
```

Comment:
It seems that it should be

```
"official_name": "Pompes Funèbres Générales",
"name": "PFG",
```

according to other data and discussion.

Reply:
I merged and did this..